torch/csrc/distributed/c10d/init.cpp

```diff
@@ -41,6 +42,20 @@ PyObject* c10d_init(PyObject* _unused) {
 
   auto module = py::handle(c10d_module).cast<py::module>();
 
+  shared_ptr_class_<::c10d::Reducer>(module, "Reducer")
+      .def(py::init<
+           std::vector<std::vector<torch::autograd::Variable>>,
+           std::shared_ptr<::c10d::ProcessGroup>>())
```

Comment:
The default process group is a Python only thing, so it needs to be passed anyway. In the DDP wrapper it will use the default one if it isn't specified.